lang-portal/frontend/.env.example

```
@@ -6,11 +6,12 @@ NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_...
 CLERK_SECRET_KEY=sk_test_...
 GO_BACKEND_URL=http://localhost:8080
 NEXT_PUBLIC_APP_ENV=development
+NEXT_PUBLIC_SEO_BASE_URL=https://sorami.aljufairi.org
 NEXT_PUBLIC_VAPI_PUBLIC_KEY=41b8e3fe-53c2-4eb2-8d8c-fdb5675ec82a
 NEXT_PUBLIC_POSTHOG_KEY=phc_2kz11HzsCgyRqaSsIv9LmKHiptzjwBlSFeJRow8x4mA
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_KEY key should go before the NEXT_PUBLIC_SEO_BASE_URL key

(UnorderedKey)